src/components/modules/PageTitle/PageTitle.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import './PageTitle.css';
+
+const PageTitle = props => {
```

Comment:
Do we make another file for even a thing such as page title?

Reply:
Ya `component`
The rationale behind this is, all page title looks same - standardization